frontend/src/stories/ActivityDashboardBarGraph.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import ActivityDashboardBarGraph from "../components/pages/UserActions/ActivityDashboardBarGraph";
```

Comment:
There is a new standard around StoryBook. It's worth following otherwise someone will just need to clean this up later.